parsons/local_server.py

```diff
@@ -133,7 +133,7 @@ def submit():
     problem_name = request.form['problem_name']
     submitted_code = request.form['submitted_code']
     parsons_repr_code = request.form['parsons_repr_code']
-    fname = f'{PARSONS_FOLDER_PATH}/{problem_name.lower()}.py'
+    fname = problem_name_to_file(problem_name, 'py')
```

Comment:
I moved logic into a file for testability (since this file is so hard to test right now, given the complexity of the okclient import).